test/rpc/cart_rpc_one_node.py

```diff
@@ -62,7 +62,7 @@ def test_cart_rpc(self):
         clicmd = self.utils.build_cmd(self, self.env, "cli", False, urifile)
 
         self.utils.launch_srv_cli_test(self, srvcmd, clicmd)
-        self.utils.log_check(self)
+        #self.utils.log_check(self)
```

Comment:
did you mean to leave these commented?

Reply:
yes. after the application was modified to spawn multiple threads, the log parsing is having issues with multiple thread ids performing actions. as such log check for now is disabled, with vikram assigned to look into it